dist/s_docs

```diff
@@ -225,6 +225,7 @@ EOF
 
     # Copy any needed javascript
     cp ../src/docs/js/* ../docs/
+    cp ../src/docs/*.html ../docs/
```

Comment:
There have never been any pre-existing HTML files in the doc directory before. Why is this necessary?

Reply:
Because the call graphs are rather big, these HTML pages also include a minimal script to pan and zoom it (similar to maps). I thought, it's a good visual illustration of the subject.

Reply:
Thanks for the explanation.